src/mscorlib/shared/System/Buffers/TlsOverPerCoreLockedStacksArrayPool.cs

```diff
@@ -289,6 +433,54 @@ public T[] TryPop()
                 Monitor.Exit(this);
                 return arr;
             }
+
+            public void Trim(uint tickCount, int id, MemoryPressure pressure)
+            {
+                if (_count == 0)
+                    return;
+
+                Monitor.Enter(this);
+                uint trimTicks = pressure == MemoryPressure.High ? StackHighTrimAfterTicks : StackTrimAfterTicks;
+                if (_count > 0 && _stockTicks > tickCount || (tickCount - _stockTicks) > trimTicks)
+                {
+                    // We've wrapped the tick count or elapsed enough time since the
```

Comment:
The overflow from uint.MaxValue to a lower number looks like it's handled correctly  (`0x10ub` - `0xF0ub` is `0x20ub`).  The only thing that isn't handled correctly is getting called after "49.7 days" + 1 tick (which will diff as 1 tick).

So I'm not really sure what the "wrapped the tick count" refers to.

Reply:
I was referring to that overflow